docs/products/kafka/kafka-connect/howto/debezium-source-connector-mysql.rst

```diff
@@ -0,0 +1,146 @@
+Create a Debezium source connector for MySQL
+==================================================
+
+The MySQL Debezium source connector extracts the changes committed to the database binary log (binlog), and writes them to an Apache Kafka® topic in a standard format where they can be transformed and read by multiple consumers.
+
+.. _connect_debezium_mysql_schema_versioning:
+
+Schema versioning
+-----------------
+
+Database table's schema can evolve over time by adding, modifying or removing columns. The MySQL Debezium source connector keeps track of schema changes by storing them in a separate "history" topic that you can setup with dedicated ``history.*`` configuration parameters.
+
+.. Warning::
+
+    The MySQL Debezium source connector ``history.*`` parameters are not visible in the list of options available in the `Aiven Console <https://console.aiven.io/>`_ but can be inserted/modified by editing the JSON configuration settings in the **Connector configuration** section.
+
+
+.. _connect_debezium_mysql_source_prereq:
+
+Prerequisites
+-------------
+
+To setup a Debezium source connector pointing to MySQL, you need an Aiven for Apache Kafka service :doc:`with Kafka Connect enabled <enable-connect>` or a :ref:`dedicated Aiven for Apache Kafka Connect cluster <apache_kafka_connect_dedicated_cluster>`. 
+
+Furthermore you need to collect the following information about the source MySQL database upfront:
+
+* ``MYSQL_HOST``: The database hostname
+* ``MYSQL_PORT``: The database port
+* ``MYSQL_USER``: The database user to connect
+* ``MYSQL_PASSWORD``: The database password for the ``MYSQL_USER``
+* ``MYSQL_DATABASE_NAME``: The database name
+* ``SSL_MODE``: The `SSL mode <https://dev.mysql.com/doc/refman/5.7/en/connection-options.html>`_
+* ``MYSQL_TABLES``: The list of database tables to be included in Apache Kafka; the list must be in the form of ``schema_name1.table_name1,schema_name2.table_name2``
+* ``APACHE_KAFKA_HOST``: The hostname of the Apache Kafka service, needed when storing the :ref:`schema definition changes <connect_debezium_mysql_schema_versioning>`
+* ``APACHE_KAFKA_PORT``: The port of the Apache Kafka service, needed when storing the :ref:`schema definition changes <connect_debezium_mysql_schema_versioning>`
+* ``SCHEMA_REGISTRY_PORT``: The Apache Kafka's schema registry port, only needed when using Avro as data format
+* ``SCHEMA_REGISTRY_USER``: The Apache Kafka's schema registry username, only needed when using Avro as data format
+* ``SCHEMA_REGISTRY_PASSWORD``: The Apache Kafka's schema registry user password, only needed when using Avro as data format
+
+
+.. Note::
+
+    If you're using Aiven for MySQL and Aiven for Apache Kafka the above details are available in the `Aiven console <https://console.aiven.io/>`_ service Overview tab or via the dedicated ``avn service get`` command with the :ref:`Aiven CLI <avn_service_get>`.
+
+Setup a MySQL Debezium source connector with Aiven Console
```

Comment:
One of these words is probably not in our accepted list when first letter is capital, could you double check, so that we remove the warning?